.gitignore

```diff
@@ -1,2 +1,5 @@
 vendor
 composer.lock
+.idea
```

Comment:
Please don't add IDE-specific lines to the project's `.gitignore`. These files should be in your own user's `.gitignore`.